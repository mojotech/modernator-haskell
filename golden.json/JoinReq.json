{
    "seed": 3.187402296249121406e18,
    "samples": [
        {
            "questionerName": "\u0013\u001e\u000c\u0001à\\íS9\r\u0010ÛCv\u0010ö"
        },
        {
            "questionerName": ""
        },
        {
            "questionerName": "\u0011fÈ=K*ZpÎ"
        },
        {
            "questionerName": "÷\u0002\u0017òÑ@\u00193\u0012rOâ\u000bÖ\u00101g9\u0018«`h\u0004(6N"
        },
        {
            "questionerName": "(ø<\u0011I\u000bR\u0006C*~3?z9\u0015\u0006\u0004\u0016í\u0014L5G\u0017\u001bHB"
        },
        {
            "questionerName": "\u0002n7@\u000c\u0008`Y¶ }O"
        },
        {
            "questionerName": "(F _Ò\u0008ê{NµG,){Ê\u001b\u0000\u0002!s\u0002û¤c\u0006Üº"
        },
        {
            "questionerName": "ûô9"
        },
        {
            "questionerName": ""
        },
        {
            "questionerName": ".\r[\u0001\u0019g"
        },
        {
            "questionerName": "R«l^ÿ\u001b¥£ØÔ\u001da^>,!ìg\t2\u0010Y\"\"&¥<"
        },
        {
            "questionerName": "z\u001dRüQI0WKU8\u0010R\u0002"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": null
        },
        {
            "questionerName": ""
        },
        {
            "questionerName": "s°K°\u0017H±=>6«\u0002\u000fÅî6wV6<¬e"
        },
        {
            "questionerName": "\u0016-¶"
        },
        {
            "questionerName": ""
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "@·ùlM"
        },
        {
            "questionerName": "\u00110(__UK"
        },
        {
            "questionerName": "\n-\nÿÝ?äs(7nÜ\u0016=5úkg¼\u0011"
        },
        {
            "questionerName": "\u0012T\t%\u0019FkV$#I\u001at¥l\u000b\u000b,!"
        },
        {
            "questionerName": "Æ\u0001µ\u00122\u000eî*â"
        },
        {
            "questionerName": "\u0000}/í\tV`\""
        },
        {
            "questionerName": "<Þ¶Zðg¬¬Ù%6g\u0014G\t\u0016\u001d]È\u0005\n'jî\n"
        },
        {
            "questionerName": "\u00112ûÿ=zXÁ\u001b9n!4NHkKRZyÀ\u0015"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "%y1th\r\u0016\u000cmx\u0012LX\u0013\u0000X3|$hQ"
        },
        {
            "questionerName": "\r|\u00100®"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "D\u0003þ-2·èlmyLÏU®­'r`Õ"
        },
        {
            "questionerName": "\\|_\u000bì2+\u000cN\u0017Fêt-«6Du\u0002\u0007\u000båTt"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "Õ\né0*ÂC¡f\n*²z°>\u0011kå'\u0010úo\u000f"
        },
        {
            "questionerName": "*sï\u0005Ü½.7~Á|2n`$8h\u0002\u0013ÿËXov<\u001b\u0003zV"
        },
        {
            "questionerName": "<mk\u0014Dz*Fãk,~\u0007Í9O:"
        },
        {
            "questionerName": "QA¯Åxb}y¾\u0015Á\u0006ÓRq"
        },
        {
            "questionerName": "\u000b[9\u0002\u0015GÄT\u000e§v\u0005iL\u0018\u0002\u001b"
        },
        {
            "questionerName": "ST×3r.\u001dN\u0006I;­;\u000eúGFHê}9e\"V"
        },
        {
            "questionerName": "ñPp\u0011\u0015:t0l>+ÄPLé\u0016\u0001H"
        },
        {
            "questionerName": "Nx\u0014"
        },
        {
            "questionerName": "UI3TÚ\u0006DG"
        },
        {
            "questionerName": ".wH\u0019_¿\u001f{§×$¤\u0004:K8"
        },
        {
            "questionerName": "üûßI:«m\u00169\u0001O[Ïg\u000bé\u001c4I\u0002«jF"
        },
        {
            "questionerName": "N"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "µ\u001c\u0014\u0012"
        },
        {
            "questionerName": "\u0018¤}&.Fdç+)Zw"
        },
        {
            "questionerName": "c\u0014\u0005Jd2wlQt@\u0006ö\u001d\u0017x~B"
        },
        {
            "questionerName": "f\u0008é\u0000à\u0016Iä\u0006D¡)"
        },
        {
            "questionerName": "\u000c\u0011Ã:¥½{qj\u0007<Wt|ûyu@dr"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "\u0003\u000f"
        },
        {
            "questionerName": ",\u001c"
        },
        {
            "questionerName": ""
        },
        {
            "questionerName": "\u0011ãs\r"
        },
        {
            "questionerName": ""
        },
        {
            "questionerName": "êï\u0016É0¥À\u0005\u0017g\u0017D¼?"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "Gñ\u001fa}\u001a¬6':Ý FiZ\u0012Tw\u0004´\"\u0015Y\u001e"
        },
        {
            "questionerName": "ùz6jÊ(A\u0016ÛSf\u0013ÄI~)<y\t\"¨GàZ"
        },
        {
            "questionerName": "khh¨(NêC\u0017TqQqMvh\u001f[óî\"TO+#\n"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "<ø%'"
        },
        {
            "questionerName": "5ÇPzn£:é&\u0002"
        },
        {
            "questionerName": "0\u0007\tÈS\u000eQB"
        },
        {
            "questionerName": "k6Ô~&qûD3!\u0002*µ'\u0008\u0013e¬ÆK1bmqTä\u000f\u001f\u0013"
        },
        {
            "questionerName": ".T3|mk\u00050üTu\u0006k\u0015tRõ\u000bÊ\u0012}"
        },
        {
            "questionerName": "tJ&G..i/»U<Ô V\u0011üseùö"
        },
        {
            "questionerName": "S\u000eIq<{+\\cV\u001b]ÆNð\")"
        },
        {
            "questionerName": ">\u0012Lt5~\tz\u0013V\u0004]ùqpYZN?mêNH\u0007Iµj"
        },
        {
            "questionerName": " s\u00175\r?"
        },
        {
            "questionerName": "¸"
        },
        {
            "questionerName": "«M\u0001"
        },
        {
            "questionerName": "ÂµAb©\u0010ß\u001b#Xx,/ÞQ"
        },
        {
            "questionerName": "#\u001eï\u001e"
        },
        {
            "questionerName": "}øH\u001bôáJ'\u0005ª\u0016"
        },
        {
            "questionerName": "u3\u00163Üf"
        },
        {
            "questionerName": "o\u0010±RLãº"
        },
        {
            "questionerName": "Ã\u0016\u0003ç=z\u001ca#c\u0013D\u0006i)"
        },
        {
            "questionerName": "\u001d\u0007S]W{b)?"
        },
        {
            "questionerName": "A\u001d\rg5W\u0017\t\u0014ViODc\u0006K:\u000b=n\u001a\u001dô(\u000fW"
        },
        {
            "questionerName": "UC9\u0015CLu\u0001"
        },
        {
            "questionerName": "\u0002¶}.]N}0]\"\u0010\u0004ø,,ª"
        },
        {
            "questionerName": "_JMj\u0019mK;Íw\u0013LJj¾\u0004"
        },
        {
            "questionerName": "\u0019«t%\u0011\u001cKx#x/"
        },
        {
            "questionerName": "\u001f\n\u001b(\u0013\u0015Õ]\u0016'ì\\fWe"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "=\u0018\\Âz\u0019U\u0001á\u0019s9¿8wtý83#9\u0010\n{N\u0017\u0012"
        },
        {
            "questionerName": ",Ov\u000f\u001e|W"
        },
        {
            "questionerName": "0m"
        },
        {
            "questionerName": "\u0015\u000c\u001bQa1Jb.WT\u0014\u0004"
        },
        {
            "questionerName": "\u0019\u0013Nh6B\u0011ã¹Õjç1g|l"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": ",ùWTiI\u0017·"
        },
        {
            "questionerName": "~tí\u0004|z&~{\u0012 8^\u001cè\u0019n1\rg!FId "
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "pÕ"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "KÚU\u0000\u0002\u0015\u000fØNQ^^¦&Q&æ"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "\u0019\u0002ÙÊl\u0000A2%Q{8_N"
        },
        {
            "questionerName": "õTb\u0000i`<\u0000\u001e5c!\u0014V\u0013\r\u0002/2ê\u000e."
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "$~\u001d\u001e"
        },
        {
            "questionerName": ",Mö\u001e^F>qï%±+q\u000bC7"
        },
        {
            "questionerName": "Qª"
        },
        {
            "questionerName": "x¡\u001aNP2ApV\r<1\u00154ól/H\u001c"
        },
        {
            "questionerName": "%}&;1jqEtE\u0008)'\u0017\u00041j{'aT]"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "*\u0005"
        },
        {
            "questionerName": "©2d\u001d\u0017jq9\u000cN$¢Ê"
        },
        {
            "questionerName": ""
        },
        {
            "questionerName": "W\u0019Ò|YmÞ[(\u0017W?sOg-"
        },
        {
            "questionerName": "?\u0010à"
        },
        {
            "questionerName": "\u0011\u001a[\u0019jTK­<BP"
        },
        {
            "questionerName": "m~Gót!\u001f^(\u000bû\u001dr\tü>PW,/\u000cÒ3ÏU"
        },
        {
            "questionerName": "\u000eG)\u0016ºW\\D\u00183xB\rEGn"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "\u001aã\u001e"
        },
        {
            "questionerName": "\u0007C\u00162j"
        },
        {
            "questionerName": "oE\u0012H½\u0004\u0012x-É2¸üÆ37%"
        },
        {
            "questionerName": "\u000ctDU©àÑ"
        },
        {
            "questionerName": "2\u000bz>\u0007X2,Ó\u000f"
        },
        {
            "questionerName": "\u0014X6ls¡H\u0007`xR\u001c\u001a\u0016&x\u001abIé"
        },
        {
            "questionerName": "&*¤@\u0000ò]Í\rt\u0003\u001d×VmtdÔ`N"
        },
        {
            "questionerName": "\t]/Î\nÐ\u0002\u0003à\u0002C<%\u000b \u0001}ÀvKR]"
        },
        {
            "questionerName": "wT¨N7'\u0007'two=(o\u001cd\u000fJ\t²m\u001d\u0012Âüé-l/æ"
        },
        {
            "questionerName": "8Xèo¼\u001fÏ5JLl$A\ro\u0006"
        },
        {
            "questionerName": "t\u0012[5OäM\u0013D³/ã\u001f?\u001eû\u0019\"$"
        },
        {
            "questionerName": "K"
        },
        {
            "questionerName": "eznQå\u0007KIúj;rZ\u001cOx2'"
        },
        {
            "questionerName": "nW\n\u00080\u0013c50./Bµ"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "RF.FT\u0013,\u0011®;ðïe"
        },
        {
            "questionerName": "\u00171Z\u0012\rCÙZ-\\%|~\u000c\rP"
        },
        {
            "questionerName": "?ò%^W^"
        },
        {
            "questionerName": "\u0012¡R:c\u0001³"
        },
        {
            "questionerName": "Z\u0013\u0004w!BJóÄbé`>ÙÝjO37fd\u0018}÷\u0014Í"
        },
        {
            "questionerName": "y\u001a\u0010\u0005k\u0001è\u0018\u0011^+\u0001\u0016\u000bD\u001b\n\u0015]ÅFBÂb"
        },
        {
            "questionerName": "}ÉPÃcq\u001fJ=e"
        },
        {
            "questionerName": "\u0015\r3K\u001dÍ\u0004EX\u0017$ò\u0014>_-\u001c"
        },
        {
            "questionerName": "]´"
        },
        {
            "questionerName": "£q-Y»ü"
        },
        {
            "questionerName": "S\u001fÑOÛ.\u00126à\n"
        },
        {
            "questionerName": "H¢ÐX²iv\u0000p"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "jH=%'1!<\nEw)-ó\u000fk\u0013{òNu"
        },
        {
            "questionerName": "Û"
        },
        {
            "questionerName": "9Q©.fJ\u0006WçÎtWB/\u0004wÐbgÑµGb¡\u001c"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "\u00153\u001e\u0005¢"
        },
        {
            "questionerName": "'pÓ\u0006ù©Èhº¶µH)û÷gcHFW©\u000b¯4"
        },
        {
            "questionerName": "ú"
        },
        {
            "questionerName": ",\\b²swÑ9\ns\u0006w*\u0011,\u0008Ðl\u0006Ê}"
        },
        {
            "questionerName": "âIGü"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "G]YÐ\u000c~/\u000c©Ti yFtLC¡ÉÖË=/"
        },
        {
            "questionerName": ")\u000bþ"
        },
        {
            "questionerName": "²\\ÎXNmó"
        },
        {
            "questionerName": "\u001dgö[\u0001Ô¨m\u000f´\"\u000b\u0016^\u0019"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "\u000f]'Yn(g\u001dD"
        },
        {
            "questionerName": ""
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "(ü\u000c[÷r¾Qà\u0002k3Õ"
        },
        {
            "questionerName": "¤.=5¸6\u001b!\u0001ô@ulÄT!g\u0002d\u001fçî;"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "Õr5º6\u0006d;iî\u0015Ñfä\"Éa­¡@"
        },
        {
            "questionerName": "_\u0000"
        },
        {
            "questionerName": "?l\u0002"
        },
        {
            "questionerName": "\u00188<FH`I*1\u0001ÜãÇ\u0017T\u001dt:'åXg×\u0002"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": ")Dp@pbø~â0\u0018/\u0008:"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "dv\u001d\u0014.8\u000bk¶VOzZ\u000e\u0007ê"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "K×RÃPæH\u0017¬ê\u0006!¡N\u000b\u001b¾<ò"
        },
        {
            "questionerName": "³7\u0005\u0001"
        },
        {
            "questionerName": ".\u0007\"\u001f\u0005~5l\u0007s\u0018Y3\u001e_e"
        },
        {
            "questionerName": "\u000cËøÄúë2-0BÏ,dP\u001f"
        },
        {
            "questionerName": "ÿ-%8Üâ¢,|o\u0017BkBD/1_\u001cc(>°9ÝÇ"
        },
        {
            "questionerName": "=ñÿ`d9\u0001S\u000e\u0000_\u0001l!\u001aaYBDêa&H[Gv4i"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": null
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "\u0001=Tô\u0001â\u0000PnE\u0001\u0014[s\u000e<¾\u0010"
        },
        {
            "questionerName": "%þ*\u0013\u0008\u000c9?"
        },
        {
            "questionerName": "9q\u0014ö\u001fFZ\u00188S\u0010Zsøf¥\u001c\u0017ÊjÈ\u000f\"\u0006\u001c/-"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "\u0017Ï\u0002VCj\u001b_¹9ÚT×\u001bICPm"
        },
        {
            "questionerName": null
        },
        {
            "questionerName": "OZ-&Fz),/="
        }
    ]
}