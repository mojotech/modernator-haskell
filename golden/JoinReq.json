{
    "seed": -2986429946544573198,
    "samples": [
        {
            "questionerName": "\u0012 KegE\u0016\u000egmD«\u0003\u0012jë"
        },
        {
            "questionerName": "\u001f¨ýF\u0017\u000eX\u0018l\u0018\r[À\u0000ió¡6}'õ."
        },
        {
            "questionerName": "q`¢'$âe\u0002V#î"
        },
        {
            "questionerName": "2ù\u001b \u001a\\jæ'qFi¶@\u000b"
        },
        {
            "questionerName": null
        }
    ]
}